test_opengraph_standards.py

```diff
@@ -0,0 +1,13 @@
+import requests
+import pytest
+from bs4 import BeautifulSoup
+
+
+class TestOpenGraphStandards:
```

Comment:
Or you can move your test method into this class and move the get_url fixture into your conftest.py